test/server/cards/01_SOR/units/CargoJuggernaut.spec.js

```diff
@@ -0,0 +1,38 @@
+describe('Cargo Juggernaut', function () {
+    integration(function () {
+        describe('Cargo Juggernaut\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['cargo-juggernaut'],
+                        groundArena: ['pyke-sentinel'],
+                    },
+                    player2: {
+                        groundArena: ['rugged-survivors']
+                    }
+                });
+            });
+
+            it('should heal base', function () {
+                this.p1Base.damage = 10;
+                this.player1.clickCard(this.cargoJuggernaut);
+                this.player1.clickPrompt('If you control another [Vigilance] unit, heal 4 damage from your base');
```

Comment:
Do you know why this prompt is happening? Since the ability is required it should happen automatically and not prompt the user. What exactly does the prompt say, and what buttons does it show?

Reply:
cargo juggernaut have also shield, I think it is to order triggers

Reply:
Makes sense. Can you add a comment noting that